ironic.conf

```diff
@@ -10,11 +10,11 @@ enabled_deploy_interfaces = direct,fake
 # NOTE(dtantsur): when changing this, make sure to update the driver
 # dependencies in Dockerfile.
 enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish,ibmc
-enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish
-enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc
-enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc
-enabled_raid_interfaces = no-raid,irmc,agent,fake
-enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake,ibmc
+enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish,idrac-wsman
```

Comment:
All occurrences of just `drac` should be removed in favour of `drac-wsman`.

Reply:
@jtwill98 @demonCoder95 can we do that?

Reply:
Just added idrac-wsman interface for RAID. Which got already approved in PR #146 but I had to create new PR for technical issues.

Reply:
I agree with @dtantsur here. ``drac`` is the deprecated name, we should use ``idrac-wsman`` here.